herbridge/main/views.py

```diff
@@ -123,11 +122,22 @@ def get_images_for_polygon(request):
     else:
         return JsonResponse(status=400, data={"message": "Missing request body"})
 
+def submit_image_for_resource(request):
+    if request.method != "POST":
+        raise Http404()
+    elif request.body:
+        response = requests.post('http://34.248.167.252/api/herbridge/put', data=request.body)
+        if response.status_code == 201:
+            return JsonResponse(response.json(), safe=False)
+        else:
+            return response
+    else:
+        return JsonResponse(status=400, data={"message": "Missing request body"})
+
```

Comment:
Any specific reasoning for why we route this through the HerBridge python backend instead of submitting directly from the JS? Just more consistent?